src/models/Character.js

```diff
@@ -14,7 +14,7 @@ module.exports = (sequelize, DataTypes) => {
     },
     name: {
       type: DataTypes.STRING,
-      allowNull: false,
```

Comment:
Oh no we can’t unset the name as we need that data to be able to handle requests that only know the character id. Since the Blizz API only allows for calls with region, realm and name present anyway, perhaps we can use those parameters as value rather than the values sent back by their API?

Reply:
That was my mistake.  I think keeping the name is fine since it's passed as a value of the call.